src/emerger.sh

```diff
@@ -19,8 +19,16 @@ printf "$LOGO"
 if [[ $(stty size | awk '{print $2}') -ge 74 ]]; then
     cat $SRC/utils/.logo
 fi
-printf "Contribute @ https://github.com/MasterCruelty/eMerger $WHALE\n"
-printf "Running on: $(uname -rs)\n\n$NORMAL"
+printf "Contribute @ https://github.com/MasterCruelty/eMerger $WHALE\nRunning on: "
+
+if [[ -f "/etc/os-release" ]]; then
+    NAME=$(cat /etc/os-release | head -n 7 | tail -n 1 | cut -c 14-)
+    printf "${NAME::-1}\n"
+else
+    printf "$(uname -rs)\n"
+fi
+
+printf "$(curl -s wttr.in/?format="%l:+%c+%t+%w+%m")$NORMAL\n\n"    # wttr.in function
```

Comment:
@archetipico how this function via curl works exactly?

Reply:
The Inquisition has arrived

Reply:
Try to print `curl -s wttr.in/?format="%l:+%c+%t+%w+%m"` on your terminal. It retrieves data from [wttr.in](wttr.in) and it displays it in your terminal

Reply:
Ok cool 🚀 tested now and it's working.
Another question then, can you say me more about the arguments you pass? For example ```%1``` or ```%c``` and the others

Reply:
Yes you can find your answer [here](https://en.wikipedia.org/wiki/RTFM).

Jk.
%l = location
%c = I suppose it stands for climate and it shows the weather emoji
%t = temperature (actual, not felt)
%w = wind
%m = Moon phase

Reply:
I added these information as comment in the source file